### DISCUSSION TITLE
Lightspeed agent.

### DISCUSSION THREAD

**Discussion by catfoolyou** [3kh0/ext-remover#586] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I have Lightspeed agent installed on my computer. Though I am 99% sure that this isn't going to help against it, does anybody know a permanent and affective way to get around it and/or get rid of it? I can't uninstall or kill the task because I do not have admin access.

Heroku, vercel, cyclic, koyeb, glitch, replit, and render are blocked, so I can't deploy or use a proxy.

**Comment by crisvisthian** [comment]
i did not work

**Reply by Aka-but-nice** [reply]
We did not work.

**Reply by DankFireR** [reply]
You did not work.

**Reply by PSYclownYT** [reply]
I seem to have malfunctions
we did yes no working

**Reply by Aka-but-nice** [reply]
😱 😱

**Reply by thatboyo** [reply]
use rigtools

**Comment by Epicminer256** [comment]
I got a interest in this, I wonder if a cookie spam might break the extension as a whole?

If I work on this after December, does anyone want to test my program?

**Reply by Aka-but-nice** [reply]
This is a no brainer, but of course cookie spamming the extension (if it has web request) will just freeze your browsing context. If he's on v115 or above then he can't use debugging tools on Chrome extensions (e.g `chrome://kill`). Therefore, this is almost impossible unless you want to cookie spam its endpoint URLs.

**Reply by Epicminer256** [reply]
What I was imagining is that I could take all enpoint urls and spam cookies with a DNS+HTTP server, but this would require a physical server. This is a method that I currently use, but I just use my Windows laptop to run it one time. But this wouldn't be great for 90% of cases. 
I have been working on [Securascramble](https://github.com/Epicminer256/Securascramble), which is a Securly unblocker that I de-obfuscated. It has a program called "insecurly" which doesn't require a fancy server or javascript bookmarklets. Would be cool getting it to work on lightspeed and other blocking software.

**Reply by Epicminer256** [reply]
The de-opfuscation proccess is not 100% done yet, but it will get finished eventually.

**Reply by Aka-but-nice** [reply]
I am aware of Insecurly.... and I already had the full unobfuscated code (as I was sent the test URL made by bypassi). Although, it has updated and is now obfuscated again so I guess you'll have to do it again (which isn't easy).

**Reply by AshtonDavies** [reply]
Too slow Joe! You can decode it with devtools, which is what I did a while back, to get the raw script. Literally not that hard if you have a brain. https://chrose.netlify.app/detail/INSECURLY/assets/insecurly.js

I did change the button ID near the bottom for my website integration, but it's pretty much the same.

**Reply by Aka-but-nice** [reply]
you can literally eval = console.log :moyai:

**Reply by AshtonDavies** [reply]
> you can literally eval = console.log :moyai:

I mean, that works too. A nice thing about devtools though is that it can give you the evaluated script rendered by Chrome if you open the button's onclick eventListener.

**Reply by Aka-but-nice** [reply]
Yeah of course

**Reply by Epicminer256** [reply]
Well if I don't have anyone to test the code with, then I prob won't work on it. I want to work with a extension that isn't Securly for a change.

**Reply by Epicminer256** [reply]
> Too slow Joe! You can decode it with devtools, which is what I did a while back, to get the raw script. Literally not that hard if you have a brain. https://chrose.netlify.app/detail/INSECURLY/assets/insecurly.js

Also yeah I did that too, Just mostly trying to take the program apart and make it more readable. Strings are encrypted on firefox's de-obfuscated version, I have most of those decrypted (lmao there is even a little among us character in there).

**Reply by Epicminer256** [reply]
actually nevermind, this repo has waay too much going on, so I am un-watching it.

**Reply by Aka-but-nice** [reply]
@Epicminer256 
LMAO bro that amongus symbol is part of `chrome.extension.setUpdateUrlData("ඞ".repeat(1024))` and is for flooding Securly's update URL, therefore preventing Securly from updating any further.

**Comment by Aka-but-nice** [comment]
> What I was imagining is that I could take all enpoint urls and spam cookies with a DNS+HTTP server, but this would require a physical server. This is a method that I currently use, but I just use my Windows laptop to run it one time. But this wouldn't be great for 90% of cases. I have been working on [Securascramble](https://github.com/Epicminer256/Securascramble), which is a Securly unblocker that I de-obfuscated. It has a program called "insecurly" which doesn't require a fancy server or javascript bookmarklets. Would be cool getting it to work on lightspeed and other blocking software.

This man stole Cookie Dough's whole flow 💀 at least credit them.

**Reply by Epicminer256** [reply]
So is zek-c not the author of securly-kill? I tried to look for the original author and he claimed to be the original author.

**Reply by Epicminer256** [reply]
I kinda want to know the OG authors, cause I tried to credit them

**Reply by Aka-but-nice** [reply]
Bro both of y'all skidded cookie dough 💀. Check in TN and go the Cookie Dough channel (located in kajigs), only there is where you'll find what you are searching for.

**Comment by py660** [comment]
Try dsf.pr.wip.la? [insert].pr.wip.la?

**Comment by sdsdsdsrj4rjj** [comment]
@catfoolyou use lightsped killer `data:text/html;charset=utf-8,<!DOCTYPE html>%0A<html lang%3D"en">%0A<head>%0A <meta charset%3D"UTF-8">%0A <meta name%3D"viewport" content%3D"width%3Ddevice-width%2C initial-scale%3D1.0">%0A <title>LightSPED Killer Agent<%2Ftitle>%0A <link rel%3D"shortcut icon" type%3D"image%2Fpng" href%3D"https%3A%2F%2Fraw.githubusercontent.com%2FBlobby-Boi%2FLightSPED-Killer-Agent%2Frefs%2Fheads%2Fmain%2Flightspeed.png">%0A <link href%3D"https%3A%2F%2Ffonts.googleapis.com%2Fcss2%3Ffamily%3DRoboto%3Awght%40400%3B700%26display%3Dswap" rel%3D"stylesheet">%0A  <style>%0A  body %7B%0A     background-color%3A %231e1f22%3B%0A  %7D%0A  <%2Fstyle>%0A <script>%0A document.addEventListener("DOMContentLoaded"%2C () %3D> %7B%0A const main %3D "https%3A%2F%2Fraw.githubusercontent.com%2FBlobby-Boi%2FLightSPED-Killer-Agent%2Frefs%2Fheads%2Fmain%2Findex.html"%3B%0A const fallback %3D "https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2FBlobby-Boi%2FLightSPED-Killer-Agent%2Findex.html"%3B%0A%0A fetch(main)%0A .then(response %3D> %7B%0A if (!response.ok) throw new Error("Main URL failed")%3B%0A return response.text()%3B%0A %7D)%0A .catch(() %3D> %7B%0A return fetch(fallback).then(response %3D> %7B%0A if (!response.ok) throw new Error("Fallback URL failed")%3B%0A return response.text()%3B%0A %7D)%3B%0A %7D)%0A .then(html %3D> %7B%0A document.open()%3B%0A document.write(html)%3B%0A document.close()%3B%0A %7D)%0A %7D)%3B%0A <%2Fscript>%0A<%2Fhead>%0A<%2Fhtml>`

**Reply by catfoolyou** [reply]
Bruh `data:text` is blocked

**Reply by schoolacc5252** [reply]
same

**Reply by BoppyCow23** [reply]
@catfoolyou wdym by blocked, does it close after opening or is it just blocked with lightsped/policys

**Reply by kiddy-gay** [reply]
Where do I input this into?

**Reply by ltorl** [reply]
you can use `data://text` instead. 

Full data URi:
`data://text/html;charset=utf-8,<!DOCTYPE html>%0A<html lang%3D"en">%0A<head>%0A <meta charset%3D"UTF-8">%0A <meta name%3D"viewport" content%3D"width%3Ddevice-width%2C initial-scale%3D1.0">%0A <title>LightSPED Killer Agent<%2Ftitle>%0A <link rel%3D"shortcut icon" type%3D"image%2Fpng" href%3D"https%3A%2F%2Fraw.githubusercontent.com%2FBlobby-Boi%2FLightSPED-Killer-Agent%2Frefs%2Fheads%2Fmain%2Flightspeed.png">%0A <link href%3D"https%3A%2F%2Ffonts.googleapis.com%2Fcss2%3Ffamily%3DRoboto%3Awght%40400%3B700%26display%3Dswap" rel%3D"stylesheet">%0A  <style>%0A  body %7B%0A     background-color%3A %231e1f22%3B%0A  %7D%0A  <%2Fstyle>%0A <script>%0A document.addEventListener("DOMContentLoaded"%2C () %3D> %7B%0A const main %3D "https%3A%2F%2Fraw.githubusercontent.com%2FBlobby-Boi%2FLightSPED-Killer-Agent%2Frefs%2Fheads%2Fmain%2Findex.html"%3B%0A const fallback %3D "https%3A%2F%2Fcdn.jsdelivr.net%2Fgh%2FBlobby-Boi%2FLightSPED-Killer-Agent%2Findex.html"%3B%0A%0A fetch(main)%0A .then(response %3D> %7B%0A if (!response.ok) throw new Error("Main URL failed")%3B%0A return response.text()%3B%0A %7D)%0A .catch(() %3D> %7B%0A return fetch(fallback).then(response %3D> %7B%0A if (!response.ok) throw new Error("Fallback URL failed")%3B%0A return response.text()%3B%0A %7D)%3B%0A %7D)%0A .then(html %3D> %7B%0A document.open()%3B%0A document.write(html)%3B%0A document.close()%3B%0A %7D)%0A %7D)%3B%0A <%2Fscript>%0A<%2Fhead>%0A<%2Fhtml>`

**Comment by sdsdsdsrj4rjj** [comment]
bookmark


On Thu, Apr 2, 2026 at 12:12 PM KIDDY ***@***.***> wrote:

> Where do I input this into?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/586?email_source=notifications&email_token=A7CY4TZUYSGOHPEHHPYQAUT4T2NQ3A5CNFSNUABIM5UWIORPF5TWS5BNNB2WEL2ENFZWG5LTONUW63SDN5WW2ZLOOQXTCNRUGI3DINRVUZZGKYLTN5XKOY3PNVWWK3TUUVSXMZLOOSWGM33PORSXEX3DNRUWG2Y#discussioncomment-16426465>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7CY4T3LGEJ2TYM6IR5WC334T2NQ3AVCNFSM6AAAAACXK27FLOVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMNBSGY2DMNI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

-- 
  **The information contained in this message is confidential and is 
intended for the addressee only. If you have received this message in error 
or there are any problems, please notify the sender immediately. The 
unauthorized use, disclosure, copying or alteration of this message is 
strictly forbidden.

--

**Comment by Vikvippro12345** [comment]
Well, u can self host a proxy or use a custom domain with vercel and other providers
